orion/controllers/announce/announceController.go

```diff
@@ -0,0 +1,107 @@
+package announce
+
+import (
+	"errors"
+	"net/http"
+	"regexp"
+	
+	"github.com/gin-gonic/gin"
+)
+
+const REGEX = `[A-Za-z]+`
+
+func GetAnnouncements(c *gin.Context) {
+	// Query Repo
+	announceList := GetAllAnnouncements()
+
+	// JSON Response
+	c.JSON(http.StatusOK, announceList)
+	return
+}
+
+func GetAnnouncement(c *gin.Context) {
+	// Incoming parameters
+	id := c.Param("id")
+
+	// Query Repo
+	announce, err := GetAnnouncementById(id)
+	if err != nil {
+		panic(err)
+	} else {
+		c.JSON(http.StatusOK, announce)
+	}
+	return
+}
+
+func CreateAnnouncement(c *gin.Context) {
+	// Incoming JSON
+	var announceJson Announce
+	c.BindJSON(&announceJson)
+
+	if err := CheckValidAnnouncement(announceJson); err != nil {
+		c.String(http.StatusBadRequest, err.Error())
+		return
+	}
+
+	// Query Repo (INSERT & SELECT)
+	err := InsertAnnouncement(announceJson)
+	if err != nil {
```

Comment:
if it looks better, you can do this:
```
if err := InsertAnnouncement(announceJson); err != nil {
}
```
It shortens the code by a line and we don't have a lingering `err` variable.